go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/cloudtrust/common-service/v2
 
-go 1.23.0
-
-toolchain go1.24.1
+go 1.24.1
```

Comment:
Is it a compulsory change ?

Reply:
Both methods will compile the module using 1.24.1, but shouldn't we specify the same version in every module? 
I may have misunderstood